upgrade-notes/backend_20250220_144154.md

```diff
@@ -0,0 +1,7 @@
+#### Elasticsearch URLs are now relative instead of absolute ([#3811](https://github.com/shopsys/shopsys/pull/3811))
+
+- Product Elasticsearch field `product_detail` has been removed, use `slug` instead, field `brand_url` has been removed, use `brand_slug` instead
+- BlogArticle Elasticsearch field `url` has been removed, use `slug` instead
+- `BrandCachedFacade` has been removed as its functionality was redundant
```

Comment:
Redundant with what? What should I use instead? Or what about checking to make sure it is cached?